README.md

```diff
@@ -21,3 +21,44 @@ To run the code:
 - cp config/config.example.json config/config.json
 - npm install
 - npm start
+
+
+Customising the registration form
+---------------------------------
+
+The registration fields are defined in the `config.json` file, in the
+`registration.fields` section.
+
+The registration form itself is defined by the theme, in a Handlebars template
+named `registration/register.hbs`. Each field object from the configuration is
+passed to this template, as `field_XXX` where XXX is the key used in the config
+file. You can set arbitrary properties on the field object and use them in this
+template.
+
+There are also some special properties for field objects:
+
+  * `type`: see below for list of types
+  * `display_name`: name shown to the user
+  * `short_display_name`: name used when listing all registrations
+  * `required`: if this true and the field has empty value when the
+    registration form is submitted, the user will see an error asking
+    them to fill in the field.
+  * `private`: if this is true, the field will be shown to anyone viewing
```

Comment:
Well, it should be that false is shown to anyone, and true is only visible to admins and the user themselves.